cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/CombinedMetricsScanner.java

```diff
@@ -105,12 +105,16 @@ public Row next() {
   @Override
   public void close() {
     v3TableScanner.close();
-    v2TableScanner.close();
+    if (v2TableScanner != null) {
```

Comment:
these scanner's extend AutoCloseable (java.lang) and they override the close to not throw exception..